indico/MaKaC/webinterface/tpls/ConfDetailsFull.tpl

```diff
@@ -63,6 +63,8 @@
   </div>
   % endif
 
+  ${ template_hook('survey-announcement', event=conf) }
```

Comment:
I think it is ok as long as we don't want to have a single `template_hook` for all the announcements within an event.